### PR TITLE
New subcommand to list invite status

### PR DESCRIPTION
**Pull request by mikejholly**
New subcommand that helps users determine the status of previously sent invites.

```
earthly preview orgs --org earthly-technologies invite ls
User Email                     Permission  Created     Accepted
mike@earthly.dev               admin       2022-07-29  No
mikejholly@gmail.com           read        2022-08-22  2022-08-22
mikejholly+test001@gmail.com   read        2022-08-22  2022-08-22
mikejholly+test0002@gmail.com  read        2022-08-24  No
mikejholly+test0003@gmail.com  admin       2022-08-24  No
mikejholly+test0004@gmail.com  read        2022-08-25  2022-08-25
```

### CODE REVIEW

**Review by alexcb — COMMENTED** [review]
I think it would be useful to also include the functionality for a user to accept one or more of these invites.

currently it's not clear how a user could act on the pending invites.

**Comment by mikejholly** [comment]
Well, we have the invite accept command: `earthly preview org invite accept <code>`. Is that what you mean?

**Comment by alexcb** [comment]
> Well, we have the invite accept command: `earthly preview org invite accept <code>`. Is that what you mean?

How does one get the code?

From a usability point of view, it would be nice to be able to accept an invite without having to go to your email.

**Comment by alexcb** [comment]
let me revisit this -- I was thinking about this from the perspective of the use accepting the invite; not from the perspective of an admin sending out the invite.

Do we have functionality to delete pending non-accepted invites?

**Comment by alexcb** [comment]
I wonder if this should (eventually) show both received and sent invites, for example:

```
earthly preview orgs --org earthly-technologies invite ls
received:
Permission     Created        Code
read           2022-07-29     abc123

sent:
User Email                     Permission  Created     Accepted
mike@earthly.dev               admin       2022-07-29  No
mikejholly@gmail.com           read        2022-08-22  2022-08-22
mikejholly+test001@gmail.com   read        2022-08-22  2022-08-22
mikejholly+test0002@gmail.com  read        2022-08-24  No
mikejholly+test0003@gmail.com  admin       2022-08-24  No
mikejholly+test0004@gmail.com  read        2022-08-25  2022-08-25
```

Something like the above would have made it more clear to me (a user seeing it for the first time). But now that I understand it's only for outgoing invites, it seems good.

File: cmd/earthly/org_cmds.go [hunk 1]
```diff
@@ -122,6 +122,14 @@ func (app *earthlyApp) orgCmdsPreview() []*cli.Command {
 					UsageText:   "earthly org invite accept <invite-code>",
 					Action:      app.actionOrgInviteAccept,
 				},
+				{
+					Name:        "ls",
+					Aliases:     []string{"list"},
+					Usage:       "List all pending and accepted invitations",
```
Comment (alexcb) [inline cmd/earthly/org_cmds.go:128]: maybe this needs to be clarified that it's only sent invites:
```suggestion
					Usage:       "List all sent invitations (both pending and accepted)",
```

**Review by alexcb — APPROVED** [review]
looks good, just a minor suggestion for usage text.

**Review by vladaionescu — APPROVED** [review]

**Comment by mikejholly** [comment]
> not from the perspective of an admin sending out the invite.

Yep, this is meant to give admins an idea of who's been invited and still needs to accept. Personally, I think accepting via email is a good start. Perhaps we can improve the non-email process later.

> Do we have functionality to delete pending non-accepted invites?

No, not at the moment. But it's not a bad idea.

**Pull request merged**